Deploy/UtilityScripts/CpqApiHelper.py

```diff
@@ -115,10 +116,79 @@ def deleteGlobalScript(self, id):
         )
         return response
 
+    # Custom Table APIS
+
+    def getAllCustomTables(self):
+        print("121")
+        headers = self.getCTHeaderJwt()
+        headers["Content-Type"] = "application/json"
+        headers["accept"] = "*/*"
+        url = self.__host + "api/custom-table/v1/customTables"
+        # tables = requests.get(url, headers=headers)
+        tables= self.testCallSuccess(
+            requests.get,
+            url,
+            headers=headers
+        )
+        print('131')
+
+        if tables.status_code == 200:
+            tables_data = tables.json()
+            return tables_data
+        else:
+            print(f"Request failed with status code: {tables.status_code}")
+
+    def updateCustomTables(self, table_name, payload):
+        data = payload
+        headers = self.getHeaderJwt()
+        headers["Content-Type"] = "application/json"
+        tableName = table_name
+        url = self.__host+ f"api/custom-table/v1/customTables/{tableName}"
+        tables= self.testCallSuccess(
+            requests.patch,
+            url,
+            headers=headers,
+            data=json.dumps(data)
+        )
+
+        print(tables)
```

Comment:
Same thing here,  for consistency, 
response = self.testCallSuccess
return response